client/my-sites/stats/stats-email-summary/index.jsx

```diff
@@ -71,6 +76,26 @@ const StatsEmailSummary = ( { period, query } ) => {
 		titleLogo: null,
 	};
 
+	const downloadCsvElement = shouldGateCsvDownloads ? (
+		<DownloadCsvUpsell siteId={ siteId } borderless />
+	) : (
+		<DownloadCsv
+			statType="statsEmailsSummary"
+			path="emails"
+			query={ query }
+			period={ period }
+			headers={ [ 'title', 'opens_rate', 'unique_clicks', 'link' ] }
+			rowModifierFn={ ( row, data ) => {
+				if ( ! row?.length ) {
```

Comment:
```suggestion
				if ( ! Array.isArray( row ) || row.length === 0 ) {
```
Could we check the argument `row` more specifically, since it is more expected to be an array of strings than a string?

Reply:
OMG, yes! Thanks!